### PR TITLE
Laboratory work #7, Pareshina Evgenia - 20FPL2 

### PR DESCRIPTION
**Pull request by eapareshina**

### CODE REVIEW

File: lab_7_llm/main.py [hunk 1]
```diff
@@ -171,6 +171,7 @@ def _infer_batch(self, sample_batch: Sequence[tuple[str, ...]]) -> list[str]:
         """
 
 
+
```
Comment (SalnikovIgor) [inline lab_7_llm/main.py:220]: Delete line

**Comment by SalnikovIgor** [comment]
Necessary implementation of loading your dataset.

**Comment by SalnikovIgor** [comment]
It's time to write the loading of your dataset.

**Comment by SalnikovIgor** [comment]
And again: it's time to write the loading of your dataset.

**Comment by SalnikovIgor** [comment]
And again: it's time to write the loading of your dataset.

**Comment by SalnikovIgor** [comment]
It's time to start doing your first lab.

**Comment by SalnikovIgor** [comment]
It's time to start doing your first lab.

**Review by SalnikovIgor — CHANGES_REQUESTED** [review]
It's time to start doing your first lab.

**Review by SalnikovIgor — CHANGES_REQUESTED** [review]
It's time to start doing your first lab.

**Review by SalnikovIgor — CHANGES_REQUESTED** [review]
It's time to start doing your first lab.

File: lab_7_llm/main.py (hunk 1 above)
Comment (eapareshina) [reply lab_7_llm/main.py:220]: Done

**Review by eapareshina — COMMENTED** [review]
comments were taken into account

File: lab_7_llm/main.py [hunk 2]
```diff
@@ -2,28 +2,21 @@
...
-
-try:
-    from pandas import DataFrame
-except ImportError:
-    print('Library "pandas" not installed. Failed to import.')
-    DataFrame = dict  # type: ignore
+import torch
+from datasets import load_dataset
+from pandas import DataFrame
+# from torch.utils.data.dataloader import DataLoader
```
Comment (SalnikovIgor) [inline lab_7_llm/main.py]: Delete or unskip

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+datasets==2.16.1
```
Comment (SalnikovIgor) [inline requirements.txt:1]: Add line `--extra-index-url https://download.pytorch.org/whl/cpu`

**Review by SalnikovIgor — CHANGES_REQUESTED** [review]

**Comment by SalnikovIgor** [comment]
Fix minor comments and start lab 8

**Comment by eapareshina** [comment]
> Fix minor comments and start lab 8

Thank you very much! Comments are already fixed

**Pull request closed without merging**